src/lib/FullpageController.svelte

```diff
@@ -81,12 +85,21 @@
     }
     const handleDragging = (event) => {
         if (dragging) {
-            fullpageScroll.set(dragStartScroll - (event.clientY - dragPosition), {
-                duration: 0
-            })
+            const distance = Math.abs(event.clientY - dragPosition);
+            const threshold = distance > (fullpage.clientHeight / dragThreshold);
```

Comment:
I think here is too much math done for event that's fired every time user drags. Especially division is pretty demanding, it would be fine to mitigate it.

Reply:
Maybe we can improve the performance by using squared distances as opposed to absolutes, for example:
```js
const distance = Math.pow(event.clientY - dragPosition, 2);
const threshold = Math.pow(fullpage.clientHeight / dragThreshold, 2);
```

This way we can avoid the square root calculation involved in computing the actual distances. The side-effect of this is maybe a few edge cases where user inputs are wrongly identified.

Another option is to use bitwise operations, an example would be:
```js
const distance= (event.clientY - dragPosition) ** 2;
const threshold = (fullpage.clientHeight / dragThreshold) ** 2;
```
This optimization works because its only interested in whether one distance is greater than another, and you don't need the actual values; however, I believe the performance gain opposed to this will not be significant as this is not a large enough calculation to warrant it.

Reply:
Thanks for trying for optimization,
but I meant so meting different, like changing semantics of dragThreshold to entirely mitigate division. Actually calculating squared distances would not speed up calculations as it is defined not by power and root, but rather as [negation of negative value](http://www.devdoc.net/web/developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs.html), and you can see it in [V8 code](https://github.com/v8/v8/blob/25933a97b1c4a4beeabc16da9694e66f37522457/src/builtins/math.tq#L37). And FYI ['**'](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Exponentiation) operator is actually not bitwise operator, but just expressive writing of power.